src/themes/colors.ts

```diff
@@ -24,6 +24,14 @@ const Colors = {
     shade800: '#303030',
     shade900: '#212121',
   },
+  toast: {
```

Comment:
why we have all different colors, ideally success, error colors should be same overall the app

Reply:
i just gave the background color for different type of toast
My toast message ui is similar to android toast ui